examples/README.md

```diff
@@ -173,13 +173,15 @@ spec:
     container:
       image: containerA
       env:
-        - "LOG_LEVEL={{workflow.parameters.log_level}}"
+        - name: LOG_LEVEL
+        - value: "{{workflow.parameters.log_level}}"
```

Comment:
This shouldn't be a separate entry in the array.
    env:
    - name: LOG_LEVEL
      value: "{{workflow.parameters.log_level}}"

Reply:
Thanks, I'll tfix thsi!